src/main/java/io/vertx/core/json/JsonObject.java

```diff
@@ -334,7 +334,7 @@ public static JsonObject of(String k1, Object v1, String k2, Object v2, String k
   public static JsonObject of(String k1, Object v1, String k2, Object v2, String k3, Object v3,
                               String k4, Object v4, String k5, Object v5, String k6, Object v6,
                               String k7, Object v7, String k8, Object v8, String k9, Object v9) {
-    JsonObject obj = new JsonObject(new LinkedHashMap<>(9));
+    JsonObject obj = new JsonObject(new LinkedHashMap<>(14));
```

Comment:
```suggestion
    JsonObject obj = new JsonObject(new LinkedHashMap<>(13));
```

https://guava.dev/releases/21.0/api/docs/src-html/com/google/common/collect/Maps.html#line.291